installer/bundle/bundle_skel.sh

```diff
@@ -610,36 +610,17 @@ pkg_upd() {
         return $?
     else
         [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
-        rpm --upgrade --replacepkgs $FORCE ${pkg_filename}.rpm
+        # Only for RedHat 7.5 call upgrade with --replacepkgs flag and only if force flag is not set
+        redhat_7.5=`cat /etc/*-release 2>/dev/null | grep -iP '.*?red.*?7\.5' 2>&1>/dev/null; echo $?`
```

Comment:
does rhel provide minor version in the `-release` file? just checked my CentOS machine and don't see it

Reply:
yes, it does.